src/Rings/primary_decomposition_helpers.jl

```diff
@@ -169,13 +169,23 @@ end
   J = saturated_ideal(I)
   res = absolute_primary_decomposition(J)
   result = []
+  if is_empty(res)
+    U = ideal(A, one(A))
+    return [(U, U, U, 0)]
+  end
+  
+  # Create the ring for the return values
   for (P, Q, P_prime, d) in res
+    R_prime = base_ring(P_prime) # the new polynomial ring
+    L = coefficient_ring(R_prime) # the new field for the result
+    A_ext, ext_map = change_base_ring(L, A) # recreate the quo-ring over that field
+    @assert coefficient_ring(base_ring(A_ext)) === L
+    help_map = hom(R_prime, A_ext, gens(A_ext); check=false)
+    help_map(first(gens(P_prime)))
```

Comment:
Is this different from this?
```suggestion
    help_map(gen(P_prime, 1))
```

Reply:
Thanks for catching this! But it is actually an unremoved line from debugging and completely useless and faulty.